### DISCUSSION TITLE
Adopting Prebuild

### DISCUSSION THREAD

**Discussion by solokhind** [infinitered/ignite#2394] [Q&A] [answered]
Hi, why doesn't ignite use [expo's prebuild feature](https://docs.expo.dev/workflow/prebuild/#pitch)?
Upgrading expo sdk and react-native is time intensive and error-prone.
I'm planning on adopting it in my project, created with ignite. Please let me know if there is anything I should be aware of before I proceed. Thanks!

**Comment by frankcalise** [comment, marked as answer]
Hey @solokhind - we're actually discussing that internally some.

There is no reason why you can't adopt prebuild now or do the Expo Dev Client. You can look to this recipe, [Pristine Expo Project](https://ignitecookbook.com/docs/recipes/PristineExpoProject/) to get you started!

**Reply by solokhind** [reply]
Thanks for your quick response! I'll go ahead and work on adopting the prebuild, I do have some of my own native code so I'll have to keep using the bare workflow.

**Reply by frankcalise** [reply]
Sounds great! If you hit any snags that we should look into upstream, feel free to make an issue and even open a PR if you can!
